src/Subscriptions/SubscriptionBroadcaster.php

```diff
@@ -86,7 +86,7 @@ function (Subscriber $subscriber) use ($root): void {
                 $data = $this->graphQL->executeQuery(
                     $subscriber->query,
                     $subscriber->context,
-                    $subscriber->args,
+                    $subscriber->context->request()->input('variables'),
```

Comment:
This seems like the wrong place to do this, the `Subscriber` is supposed to hold the serialized information that is necessary to re-execute a query. `->args` should just hold the variables?

Reply:
I think the args are used elsewhere, since we also pass in the whole Subscribe-object as an argument on `executeQuery`. What about adding variables into the Subscriber without touching args?

Reply:
You are right, `Subscriber::$args` is used in many places and is useful as is.

We can just use `$resolveInfo->variableValues` and store that in the `Subscriber` object. That is preferrable to the approach you took here, since it avoids ad-hoc reparsing of the request. The client could have sent the variables in a way that is not compatible with `->input()`, see https://github.com/laragraph/utils/blob/master/src/RequestParser.php

Reply:
I see. Thanks 👍